uast/utils.go

```diff
@@ -17,3 +17,34 @@ func Tokens(n *Node) []string {
 	}
 	return tokens
 }
+
+// This returns the cyclomatic complexity for the node. This uses the method of counting one + one
```

Comment:
Yes, checked with a Python snippet that is semantically the same as the node in the unittest, I've also checked some implementations of this and they basically do the same ([example1](https://github.com/fzipp/gocyclo/blob/master/gocyclo.go#L215) [example2](http://checkstyle.sourceforge.net/config_metrics.html#CyclomaticComplexity)).